tasks/test_syntax.js

```diff
@@ -125,7 +127,39 @@ function assertFileNames() {
 
         log('lower case only file names', logs);
     });
+}
+
+// check that all files have a trailing new line character
+function assertTrailingNewLine() {
+    var pattern = combineGlobs([
+        path.join(constants.pathToSrc, '**/*.glsl'),
+        path.join(constants.pathToRoot, 'test', 'image', 'mocks', '*')
+    ]);
+
+    var regex = /\r?\n$/;
```

Comment:
done in https://github.com/plotly/plotly.js/pull/1419/commits/f13091c5dd29ccadcb404d2a7c7975b7c1ec6341